nussknacker-dist/src/universal/conf/dev-application.conf

```diff
@@ -35,16 +35,16 @@ scenarioTypes {
   "streaming": {
     deploymentConfig: ${flinkDeploymentConfig}
     modelConfig: {
-      classPath: [ ${managementModelDir}"/managementSample.jar" ]
+      classPath: [ ${genericModelDir}"/genericModel.jar" ]
```

Comment:
but here we miss openapi.jar (and sql.jar) and its not trivial to setup this path

Reply:
It is quite trivial ;) - I've added the change after which it is available after dev run. Should be merged with: https://github.com/TouK/nussknacker-quickstart/pull/4 Take a look

Reply:
After talk with Maciek, I've copied `nussknacker.conf` content to `dev-application.conf` to make dev setup lest related to quickstart and avoid some additional code on quickstart side. We need to fix one thing in model config resolution yet, but I'll do this in separate PR